pkg/virt-controller/watch/migration.go

```diff
@@ -1397,6 +1391,18 @@ func (c *MigrationController) getNodeForVMI(vmi *virtv1.VirtualMachineInstance)
 	return node, nil
 }
 
+func (c *MigrationController) triggerAlertForHostModelUnschedulable(vmi *virtv1.VirtualMachineInstance, targetPod *k8sv1.Pod) {
```

Comment:
I would suggest naming this `alertIfHostModelIsUnschedulable` or similarly. As of now, I think this will alert always unless I look to the code.

Reply:
Nice! done